proposals/notary + tuf.adoc

```diff
@@ -0,0 +1,84 @@
+== Notary & TUF Proposal
+
+*Name of project:* Notary & TUF
```

Comment:
Ah, okay I wasn't aware that Python hadn't switched. I have actually brought up TUF to the zypper folks (our package manager), so don't think I'm not interested in it as a project. :smile_cat:

> GRPC has integrations with many languages and that seems to be nothing but a good thing.

That's not the same thing though. gRPC's multiple integrations is because in order to use gRPC with a language you have to generate client/server code from the schema. You don't need to do that with TUF, and as you said the publishing and verification can be done by either implementation.

This situation would be more like wanting to include the DBus "specification" as well as two different dbus implementations *in one proposal*. If the key benefit or differentiator is the language they're implemented in, I'm struggling to see why you need both.